apollo/src/graphql/model/modelQueries.ts

```diff
@@ -4,8 +4,13 @@ import { builder } from '../../builder.js';
 builder.queryFields((t) => ({
     listMLModels: t.field({
         type: [MLModel],
-        async resolve(root, args, ctx) {
-            const mlModel = await ObjectionMLModel.query().orderBy('dateCreated');
+        args: {
+            modelName: t.arg.string(),
+        },
+        async resolve(_root, args, _ctx) {
+            const mlModel = await ObjectionMLModel.query()
+                .where('modelName', 'LIKE', `${args.modelName || ''}%`)
```

Comment:
@StephenODea54 - If you want case insensitivity, this should use `ILIKE` instead of `LIKE`; adding a `.toLowerCase()` isn't going to affect much since Postgres has case-sensitive string comparison by default.

I also am not sure that you want to be matching on an empty string: this will return an empty set when no `modelName` argument is supplied. Instead, use
```typescript
let query = ObjectionMLModel.query()
if (modelName) {
    query.where('modelName', 'ILIKE', args.modelName)
}

const mlModel = await query;
return mlModel
```

Generally, please don't merge unreviewed commits or commits with changes requested. I'm sorry it took me a minute to respond to the updated PR; I was cleaning bathrooms

Reply:
Gotcha, my bad.